Signal/src/util/OWSBackupExportJob.m

```diff
@@ -346,7 +345,7 @@ - (void)start
         if (self.isComplete) {
             return;
         }
-        [self tryToFetchManifestWithCompletion:^(BOOL tryToFetchManifestSuccess) {
+        [self fetchAllRecordsWithCompletion:^(BOOL tryToFetchManifestSuccess) {
```

Comment:
Now that we have a local cache of "known backup fragments", we no longer need to download the "last known manifest".